### PR TITLE
Add L3AF Technical Charter

### PR DESCRIPTION
**Pull request by sanfern**
Signed-off-by: Santhosh Fernandes <santhosh.fernandes@gmail.com>

### CODE REVIEW

**Review by vmbrasseur — APPROVED** [review]
No content changes. All were purely formatting, converting the ordered lists to HTML and fixing the header levels. This will help screen readers to do the correct thing.

**Review by vmbrasseur — APPROVED** [review]
OK, final change (I think).

This one removes the PDF since the markdown will be the definitive version after this is merged. Having the PDF hanging around will just cause confusion, so cleaning that up now.

**Review by dthaler — COMMENTED** [review]
Should the PDF be linked from some page at https://l3af.io/ ?

File: docs/L3AF_technical_charter.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+This Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the
+L3AF open source project, which has been established as L3AF Project a Series of LF Projects, LLC (the “Project”).
+LF Projects, LLC (“LF Projects”) is a Delaware series limited liability company. All contributors (including
+committers, maintainers, and other technical positions) and other participants in the Project (collectively,
+“Collaborators”) must comply with the terms of this Charter.
+
+## 1. Mission and Scope of the Project
+
+<ol type="a">
+   <li>The mission of the project is to empower the Linux kernel networking eBPF community by providing a cloud and vendor agnostic platform to orchestrate eBPF programs into the kernel, thereby enabling an eco-system that delivers Kernel Function As A Service.</li>
```
Comment (dthaler) [inline docs/L3AF_technical_charter.md:14]: I agree this is what the charter says so approving, but I think the mission should be expanded beyond empowering just the Linux kernel networking eBPF community, to empowering the eBPF community as a whole (not just networking, not just Linux, not just kernel).

**Review by dthaler — APPROVED** [review]

**Review by dalalkaran — APPROVED** [review]
LGTM, let's continue to evolve the scope of this project as we move forward.

**Pull request merged**